### ISSUE TITLE
[PE-D][Tester A] New task not displaying after assign command

### ISSUE THREAD

**Issue by nus-se-script** [AY2324S2-CS2103T-T14-1/tp#141]
After running command:
`assign task/Start Project Proposal by/20-04-2024 2359 to/4`
Task is not displayed on the bottom left column, but only after clicking on the brown box that shows up.

![image.png](https://raw.githubusercontent.com/JiaXinEu/ped/main/files/fb535344-0263-47b9-af87-19f4c1383e99.png)

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: JiaXinEu/ped#4
